app/javascript/flavours/glitch/components/account.jsx

```diff
@@ -169,9 +206,14 @@ class Account extends ImmutablePureComponent {
       <div className='account'>
         <div className='account__wrapper'>
           <Permalink key={account.get('id')} className='account__display-name' title={account.get('acct')} href={account.get('url')} to={`/@${account.get('acct')}`}>
-            <div className='account__avatar-wrapper'><Avatar account={account} size={size} /></div>
-            {mute_expires_at}
-            <DisplayName account={account} />
+            <div className='account__avatar-wrapper'>
+              <Avatar account={account} size={size} />
+            </div>
+
+            <div>
+              <DisplayName account={account} />
+              <ShortNumber value={account.get('followers_count')} renderer={counterRenderer('followers')} /> {verification} {muteTimeRemaining}
```

Comment:
I need to check, but we might need to adjust this as glitch-soc offers an option to hide follower counts.

Reply:
If I'm understanding this design correctly, I believe we return a negative value when the 'hide your followers count' feature is enabled, and then conditionally handle that inline to show '-' when `account.get('followers_count') < 0`

From app/serializers/rest/account_serializer.rb:
```
def followers_count
  (Setting.hide_followers_count || object.user&.setting_hide_followers_count) ? -1 : object.followers_count
end
```

Example usage from action_bar:
`{ account.get('followers_count') < 0 ? '-' : <FormattedNumber value={account.get('followers_count')} /> }`